auditlog/models.py

```diff
@@ -208,6 +215,58 @@ def _get_pk_value(self, instance):
             pk = self._get_pk_value(pk)
         return pk
 
+    def _get_serialized_data_or_none(self, instance):
```

Comment:
Can this method still return a `None`? If not, the `none` should be eliminated from the method name.

Reply:
It still returns none.

```python
    opts = auditlog.get_serialize_options(instance.__class__)
    if not opts["serialize_data"]:
        return None
```